package.json

```diff
@@ -58,6 +58,7 @@
     "glob": "^7.1.6",
     "govuk-frontend": "^3.10.2",
     "helmet": "^4.4.1",
+    "http-status-codes": "^2.1.4",
```

Comment:
oh JavaScipt... of course there is a package:)

Reply:
Was "borrowed" from [hmcts/div-petitioner-frontend](https://github.com/hmcts/div-petitioner-frontend) since it reads better than status code numbers 😄